src/transformers/tokenization_utils_fast.py

```diff
@@ -54,6 +54,7 @@
 TOKENIZER_FILE = "tokenizer.json"
 SPECIAL_TOKENS_MAP_FILE = "special_tokens_map.json"
 TOKENIZER_CONFIG_FILE = "tokenizer_config.json"
+VOCAB_FILE = "tokenizer.model"
```

Comment:
```suggestion
TIKTOKEN_VOCAB_FILE = "tokenizer.model"
```
maybe?

Reply:
if this is used only for tiktoken let's go with TIKTOKEN_VOCAB_FILE no?

Reply:
True I can change that, but I wonder if Llama from `PreTrainedTokenizerFast` would load a `tokenizer.model` **spm** file because `PreTrainedTokenizerFast` does not include it unlike in [LlamaFast](https://github.com/huggingface/transformers/blob/d5af0f819af0b2737acbff82c7a14db70ff733a6/src/transformers/models/llama/tokenization_llama_fast.py#L34)

I can verify if we have a repo with a `tokenizer.model `spm file but no tokenizer.json / config"

Reply:
update: updated to TIKTOKEN_VOCAB_FILE